src/app/models/table-models/dimension-zone-enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum DimensionZoneEnum {
+  Rows = 'Rows',
+  Columns = 'Columns',
+  Candidates = 'Candidates'
+}
```

Comment:
What exactly is included in Candidates? Does it mean all dimensions possible and then some of them are in rows, some columns and some nowhere or these are just the ones that are not assigned to rows and columns?

Reply:
the candidates ware all the dimensions available from the previous step, the user will assign candidates to rows and columns to compose the table. And indeed, the user can pick a subset of candidates to assign to rows and columns